locale/en/invitation.po

```diff
@@ -356,4 +356,16 @@ msgid "acceptInvitation.privacyStatement.validation"
 msgstr "Please confirm that you have read and agree privacy statement"
 
 msgid "acceptInvitation.cancel.message"
-msgstr "Are you sure want to cancel accepting invitation?"
+msgstr "Are you sure you want to cancel? Canceling now will stop the role acceptance process, and you’ll need to restart from the invitation email to accept the role again.If you’re already a user, you’ll be taken back to the dashboard. If not, you’ll need to access the invitation email to start the process again."
```

Comment:
This "...again.If you’re already..." should be "...again. If you’re already..." <-- there is an extra space after . and before "if".  

Is there a mock up for this?

Reply:
yes , Please check @Devika008 comment on this https://github.com/craft-oa/gdpr-invitation/issues/68

Reply:
@ipula it looks good except for the missing space that I could see from the mock up.  I got @Devika008 to add the missing space to the [gdpr-invitation#68](https://github.com/craft-oa/gdpr-invitation/issues/68) issue as well.